env/wflow_jet

```diff
@@ -0,0 +1,21 @@
+#%Module
+
+proc ModulesHelp { } {
+   put stderr "This module loads python environement for running SRW on"
+   put stderr "the NOAA RDHPC machine Jet"
+}
+
+module-whatis "Loads libraries needed for running SRW on Jet"
+
+module load rocoto
+
+module use -a /contrib/miniconda3/modulefiles
+module load miniconda3
+
+if { [module-info mode load] } {
+  puts "conda activate regional_workflow;"
+}
+
+if {[module-info mode remove]} {
+  puts "conda deactivate;"
+}
```

Comment:
Ugh. I forgot about this problem. The modules on jet and hera won't allow you to activate the environment from the module. It's quite annoying. 

@danielabdi-noaa did you find that it works when you load it? I am happy to test, too.

Reply:
It seems to work on hera and just now tested on jet, but with a caveat...
For some reason, loading the `wflow_hera/jet` module needs to be called twice for `conda activate` to work.
The first call to `module load wflow_hera/jet` in different errors on the machines. Hera complains `conda init $SHELL` need to be called first, while jet couldn't even find `conda` command at all as shown below.
Note that after a `module purge` and executing the commands again everything goes smoothly, but I guess I need to find a solution so that it works first time.

Here is how it went on Jet for example:

```
Daniel.Abdi@fe2 ufs-srweather-app $ module use env
Daniel.Abdi@fe2 ufs-srweather-app $ module load build_jet_intel
Daniel.Abdi@fe2 ufs-srweather-app $ module list

Currently Loaded Modules:
  1) slurm/18.08.7p1    6) hpc-intel/18.0.5.274  11) png/1.6.35   16) g2tmpl/1.10.0  21) gftl-shared/v1.3.3      26) nemsiogfs/2.5.3  31) nemsio/2.5.2  36) srw_common
  2) sutils/default     7) impi/2018.4.274       12) pio/2.5.2    17) ip/3.3.3       22) yafyaml/v0.5.1          27) sfcio/1.4.1      32) w3emc/2.7.3   37) build_jet_intel
  3) cmake/3.20.1       8) hpc-impi/2018.4.274   13) esmf/8_2_0   18) sp/2.3.3       23) mapl/2.11.0-esmf-8_2_0  28) sigio/2.3.2      33) hdf5/1.10.6
  4) hpc/1.1.0          9) jasper/2.0.25         14) fms/2021.03  19) crtm/2.4.0     24) gfsio/1.4.1             29) bacio/2.4.1      34) netcdf/4.7.4
  5) intel/18.0.5.274  10) zlib/1.2.11           15) g2/3.4.3     20) upp/10.0.10    25) landsfcutil/2.4.1       30) w3nco/2.4.1      35) wgrib2/2.0.8

Daniel.Abdi@fe2 ufs-srweather-app $ module load wflow_jet
conda: Command not found.
Daniel.Abdi@fe2 ufs-srweather-app $ module list

Currently Loaded Modules:
  1) slurm/18.08.7p1    6) hpc-intel/18.0.5.274  11) png/1.6.35   16) g2tmpl/1.10.0  21) gftl-shared/v1.3.3      26) nemsiogfs/2.5.3  31) nemsio/2.5.2  36) srw_common
  2) sutils/default     7) impi/2018.4.274       12) pio/2.5.2    17) ip/3.3.3       22) yafyaml/v0.5.1          27) sfcio/1.4.1      32) w3emc/2.7.3   37) build_jet_intel
  3) cmake/3.20.1       8) hpc-impi/2018.4.274   13) esmf/8_2_0   18) sp/2.3.3       23) mapl/2.11.0-esmf-8_2_0  28) sigio/2.3.2      33) hdf5/1.10.6   38) rocoto/1.3.3
  4) hpc/1.1.0          9) jasper/2.0.25         14) fms/2021.03  19) crtm/2.4.0     24) gfsio/1.4.1             29) bacio/2.4.1      34) netcdf/4.7.4  39) miniconda3/4.5.12
  5) intel/18.0.5.274  10) zlib/1.2.11           15) g2/3.4.3     20) upp/10.0.10    25) landsfcutil/2.4.1       30) w3nco/2.4.1      35) wgrib2/2.0.8  40) wflow_jet


Daniel.Abdi@fe2 ufs-srweather-app $ module load wflow_jet
usage: conda [-h] [-V] command ...
....
(regional_workflow) Daniel.Abdi@fe2 ufs-srweather-app $ 
```